crates/abd-clam/src/chaoda/graph_scorers.rs

```diff
@@ -4,6 +4,7 @@ use std::hash::Hash;
 use crate::core::cluster::Cluster;
 use crate::core::graph::Graph;
 use distances::Number;
+use statistical::{mean, population_standard_deviation};
```

Comment:
There is a `utils` module in `abd-clam` crate and it provides functions to calculate mean, variance and std-dev. Use those instead of the `statistical` crate. Once done, move `statistical` back to a `dev-dependency` in `Cargo.toml`.